src/components/Explore/ExploreChart.tsx

```diff
@@ -176,7 +178,7 @@ const ExploreChart: React.FC<{
             <GridRows<number> scale={yScale} width={innerWidth} />
           </ChartStyle.LineGrid>
           <RectClipGroup width={innerWidth} height={innerHeight}>
-            {series.map(({ label, data, type }) => (
+            {series.map(({ label, data, type, params }) => (
               <SeriesChart
```

Comment:
nit: It seems a bit weird that we're creating multiple "charts"...  I'd expect a single **chart** with multiple **series** or similar.  So maybe `SeriesChart` should be `ChartSeries` instead or something?

Anyway, feel free to punt for now.